examples/7_train_policy_distributed.py

```diff
@@ -0,0 +1,105 @@
+"""
+This script demonstrates how to train ACT policy with distributed training on the Aloha environment
+on the Insertion task, using HuggingFace accelerate.
```

Comment:
Any chance we can do transfer instead of insertion? Insertion yields crappy results, and although I'm pretty sure it's not our fault, it could reflect poorly on us if someone doesn't know that it's crappy on original ACT as well.

Reply:
✅